src/subdomains/lightning/services/lightning-forward.service.ts

```diff
@@ -80,9 +84,17 @@ export class LightningForwardService {
     const hitId = scan.callback.split('/').at(-1) as string;
     scan.callback = LightningHelper.createBoltcardLnurlwCallback(hitId);
     scan.payLink = LightningHelper.createBoltcardPayLink(hitId);
+    scan.defaultDescription = LightningHelper.getBoltcardDefaultDescription(hitId);
     return scan;
   }
 
+  async boltcardLnurlpForward(req: Request, body: any, params: any, hitId: string): Promise<BoltcardLnurlPayDto> {
+    const payRequest: BoltcardLnurlPayDto = await this.boltcardsRequest(req, body, params);
+    if (!payRequest) throw new NotFoundException('Pay request not found');
+    payRequest.callback = LightningHelper.createBoltcardPayCallback(hitId);
+    return payRequest;
+  }
+

```

Comment:
Just a general comment on the readability of the program code:
The program code can be read more easily if a few blank lines are used. This improves the quality of the software.

Example:
```
const payRequest: BoltcardLnurlPayDto = await this.boltcardsRequest(req, body, params);
if (!payRequest) throw new NotFoundException('Pay request not found');

payRequest.callback = LightningHelper.createBoltcardPayCallback(hitId);

return payRequest;
```